numba/parfors/parfor.py

```diff
@@ -121,20 +124,50 @@ class internal_prange(object):
     def __new__(cls, *args):
         return range(*args)
 
+def _datetime_minimum(a, b):
```

Comment:
I would move these implementations out of this file. I think it would be better if you placed it in `npdatetime_helpers.py`